src/plugins_types.c

```diff
@@ -467,7 +467,12 @@ ly_type_validate_patterns(struct lysc_pattern **patterns, const char *str, size_
             return ly_err_new(err, LY_EMEM, 0, NULL, NULL, LY_EMEM_MSG);
         }
 
-        rc = pcre2_match(patterns[u]->code, (PCRE2_SPTR)str, str_len, 0, PCRE2_ANCHORED | PCRE2_ENDANCHORED, match_data, NULL);
+
+        match_opts = PCRE2_ANCHORED;
+#ifdef PCRE2_ENDANCHORED
```

Comment:
Please add a short comment about the version information here for any future changes. Something like `PCRE2_ENDANCHORED was added in PCRE2 version 10.30` or whatever is the exact version.

Reply:
Sure, and yes it was added in 10.30 :)